services/firebase.service.js

```diff
@@ -0,0 +1,13 @@
+import * as firebase from "firebase";
+import secrets from "../secrets";
+
+// Initialize Firebase
+const firebaseConfig = secrets.firebase;
```

Comment:
Reminder to put the secrets in our confidential communication channels.

Reply:
Absolutely!